vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
-# Entando 6.x Release Notes 
+
+# Entando 7.0.0 Release Notes
```

Comment:
Let's keep this as 7.0. Usually MAJOR.MINOR is sufficient, 6.3.2 was an anomaly as we discussed. If we have release notes specific to 7.0.1 or other patches we can revise this - or those notes may be delivered to subscribed users via the customer portal.